src/applications/hca/actions.js

```diff
@@ -1,15 +1,17 @@
 import appendQuery from 'append-query';
 import { apiRequest } from 'platform/utilities/api';
 
-export const SUBMIT_ID_FORM_STARTED = 'SUBMIT_ID_FORM_STARTED';
-export const SUBMIT_ID_FORM_SUCCEEDED = 'SUBMIT_ID_FORM_SUCCEEDED';
-export const SUBMIT_ID_FORM_FAILED = 'SUBMIT_ID_FORM_FAILED';
+export const FETCH_ENROLLMENT_STATUS_STARTED =
```

Comment:
Renaming these actions since they will be used by both the existing ID form page as well as the HCAEnrollmentStatus component.